src/components/Charts/ModelChart.js

```diff
@@ -49,35 +50,11 @@ const ModelChart = ({
     },
     visible: !forCompareModels,
     condensedLegend: {
-      bgColor: projections.getChartSeriesColorMap().limitedActionSeries,
+      bgColor: COLORS.LIMITED_ACTION,
     },
     legendIndex: 2,
   };
 
-  const socialDistancing = {
-    className: 'social-distancing',
-    name:
-      currentIntervention === INTERVENTIONS.SHELTER_IN_PLACE
-        ? formatIntervention(INTERVENTIONS.SHELTER_IN_PLACE, ' (lax)')
-        : formatIntervention(INTERVENTIONS.SOCIAL_DISTANCING),
-    type: projection.isInferred ? 'spline' : 'areaspline',
-    data: data[1].data,
-    marker: {
-      symbol: 'circle',
-    },
-    condensedLegend: {
-      condensedName:
-        currentIntervention === INTERVENTIONS.SHELTER_IN_PLACE
-          ? condensedFormatIntervention(
-              INTERVENTIONS.SHELTER_IN_PLACE,
-              ' (lax)',
-            )
-          : condensedFormatIntervention(INTERVENTIONS.SOCIAL_DISTANCING),
-
-      bgColor: projections.getChartSeriesColorMap().socialDistancingSeries,
-    },
-  };
-
```

Comment:
This was unused except it sometimes showed up in the embed legend (even though it shouldn't).